ts/features/wallet/bancomat/component/bancomatCard/BaseBancomatCard.tsx

```diff
@@ -118,7 +119,14 @@ const BaseBancomatCard: React.FunctionComponent<Props> = (props: Props) => {
     BASE_IMG_H,
     props.abi?.logoUrl
   );
-
+  const isBancomatExpired = fromNullable(props.expiringDate)
+    .chain(ed =>
+      isExpired(
+        ed.getMonth() + 1, // getMonth() is an integer between 0 and 11
+        ed.getFullYear()
+      )
+    )
+    .getOrElse(false);
```

Comment:
The same of the previous comment

Reply:
https://github.com/pagopa/io-app/pull/2982#discussion_r616576915